modules/machine-api-overview.adoc

```diff
@@ -27,9 +27,11 @@ Machine sets:: `MachineSet` resources are groups of compute machines. Compute ma
 [WARNING]
 ====
 Control plane machines cannot be managed by compute machine sets. 
+
+Control plane machine sets provide management capabilities for supported control plane machines that are similar to what compute machine sets provide for compute machines. 
+
+For more information, see “Managing control plane machines".
```

Comment:
Machine management > Creating compute machine sets > Creating a compute machine set on AWS > [Machine API overview](https://53397--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-aws.html#machine-api-overview_creating-machineset-aws)

Reply:
Just add this bit I think